content/en/tracing/service_catalog/troubleshooting.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Troubleshooting Service Catalog
+kind: documentation
+further_reading:
+- link: "/tracing/service_catalog/setup/"
+  tag: "Documentation"
+  text: "Setting Up Service Catalog"
+---
+
+
+
+## APM-instrumented services not appearing
+
+If services that you know are instrumented for APM are not appearing in the Service Catalog list, it's likely because they have not been emitting performance data in the past hour for the selected `env` (or any Primary Tag values of your choosing) or [Secondary Primary Tag][1]. To confirm, on the Performance tab, hover over the columns where you expect the performance metrics to appear and see instructions on which environments the services are active. 
```

Comment:
```suggestion
If services that you know are instrumented for APM are not appearing in the Service Catalog list, it's likely because they have not been emitting performance data in the past hour for the selected `env` (or any Primary Tag values of your choosing) or [Secondary Primary Tag][1]. To confirm, on the **Performance** tab, hover over the columns where you expect the performance metrics to appear and see information on which environments the services are active. 
```